src/api/app/models/review.rb

```diff
@@ -66,6 +66,8 @@ class NotFoundError < APIError
   before_validation :set_reviewable_association
   after_commit :update_cache
 
+  delegate :notifiable_projects, to: :bs_request
```

Comment:
Why?

Reply:
Explained before: creating the `notifiable_projects` method in a review, together with the `notifiable_projects` in `comment` and `bs_request` allows us to retrieve the projects of a notifiable, independant of what notifiable_type it is.